independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -330,21 +424,224 @@
+                },
+                "heading_1": {
+                  "type": "object",
+                  "properties": {
+                    "rich_text": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "text": {
+                            "type": "object",
+                            "properties": {
+                              "content": {
+                                "type": "string",
+                                "description": "Text content. This field contains the actual content of your text and is probably the field you'll use most often.",
+                                "title": "Content"
+                              }
+                            },
+                            "description": "text",
+                            "title": "Text"
+                          }
+                        }
+                      },
+                      "description": "rich_text",
+                      "title": "Heading 1"
+                    },
+                    "color": {
+                      "type": "string",
+                      "description": "Color of the block.",
+                      "title": "Color"
+                    }
+                  },
+                  "description": "Heading one block objects contain the Rich_text object within the heading_1 property",
+                  "x-ms-visibility": "advanced"
+                },
+                "heading_2": {
+                  "type": "object",
+                  "properties": {
+                    "rich_text": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "text": {
+                            "type": "object",
+                            "properties": {
+                              "content": {
+                                "type": "string",
+                                "description": "Text content2. This field contains the actual content of your text and is probably the field you'll use most often.",
+                                "title": "Content"
+                              }
+                            },
+                            "description": "text",
+                            "title": "Text"
+                          }
+                        }
+                      },
+                      "description": "rich_text",
+                      "title": "Heading 2"
+                    },
+                    "color": {
+                      "type": "string",
+                      "description": "Color of the block.",
+                      "title": "Color"
+                    }
+                  },
+                  "description": "Heading one block objects contain the Rich
+_text within the heading_1 property",
```

Comment:
@sekharmalla It is a syntax error, please remove the new line from the description value.

Reply:
I have Updated the syntax error. @sriyen-msft Thanks.